dg/commands/cpd4/install.py

```diff
@@ -85,14 +97,28 @@ def install(
             abort=True,
         )
 
+    cloud_pak_for_data_storage_vendor = (
+        CloudPakForDataStorageVendor[storage_vendor.lower()] if storage_vendor is not None else None
+    )
+
+    if (storage_class is not None) and (storage_vendor is not None):
```

Comment:
We will also not need handling of such options in each command if we extend the Option to `NotRequiredIf` class as suggested in the link above.